### PR TITLE
adding product section

### PR DESCRIPTION
**Pull request by genseral**
I added the product section with a dummy description!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h2>About Us</h2>
             <p>Transcality is a leading company in the tech industry, specializing in helping businesses achieve their digital transformation goals. Our team of experts provides innovative solutions tailored to each client's unique needs.</p>
         </section>
 
+         <section id="services">
```
Comment (transcalitytest) [inline index.html:28]: can you please change the id to products

File: index.html [hunk 2]
```diff
@@ -25,6 +25,11 @@ <h2>About Us</h2>
...
+            <h2>Products</h2>
+            This is our product section ... And all our products are free!
```
Comment (transcalitytest) [inline index.html:30]: Please add an actual product description

**Review by transcalitytest — CHANGES_REQUESTED** [review]
Please make this changes! Thanks

File: index.html (hunk 1 above)
Comment (genseral) [reply index.html]: did that thanks

File: index.html (hunk 2 above)
Comment (genseral) [reply index.html]: added that!

**Review by transcalitytest — APPROVED** [review]
lgtm!

**Pull request merged**